src/main.cpp

```diff
@@ -69,7 +70,9 @@ int main(int argc, char **argv) {
 #ifdef _WIN32
 	SetCurrentProcessExplicitAppUserModelID(MAKE_WSTR(PROGRAM_AUTHOR) L"." MAKE_WSTR(PROGRAM_NAME));
 #endif
+	Fl::keyboard_screen_scaling(0);

```

Comment:
`Fl::keyboard_screen_scaling(0);` disables the new FLTK window scaling keyboard shortcuts Ctrl -/+ since Ctrl + is already in use by the magnifying glass icon.

FLTK's high DPI rendering support is still in effect if the display is already scaled by the OS instead of by the user.